packages/build/src/error/parse/parse.js

```diff
@@ -41,7 +41,17 @@ const getFullErrorInfo = function ({ error, colors, debug }) {
 const parseErrorInfo = function (error) {
   const { message, stack, ...errorProps } = normalizeError(error)
   const [errorInfo, errorPropsA] = getErrorInfo(errorProps)
-  const { type, severity, title, group, stackType, locationType, showErrorProps, rawStack } = getTypeInfo(errorInfo)
+  const {
+    type,
+    severity,
+    title,
+    group,
+    stackType,
+    locationType,
+    showErrorProps,
+    showInBuildLog,
```

Comment:
```suggestion
    showInBuildLog = true,
```

This would not avoid to specify `true` in `error/type.js`, and make it clear `true` is the recommended case.

Reply:
Addressed via https://github.com/netlify/build/pull/2439/commits/da3e0646889115b73764ee5bb1fe41132634d20a

Reply:
:+1: 

I think you might have forgotten to add the `= true` in this commit.

Reply:
🤦 missing a `:wq`